examples/guestbook/all-in-one/guestbook-all-in-one.yaml

```diff
@@ -16,26 +16,27 @@ spec:
     tier: backend
     role: master
 ---
-apiVersion: v1
-kind: ReplicationController
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
   name: redis-master
   # these labels can be applied automatically 
   # from the labels in the pod template if not set
-  labels:
-    app: redis
-    role: master
-    tier: backend
+  # labels:
+  #   app: redis
+  #   role: master
+  #   tier: backend
 spec:
   # this replicas value is default
   # modify it according to your case
   replicas: 1
```

Comment:
ditto https://github.com/janetkuo/kubernetes/blob/automated-cherry-pick-of-%2323408-upstream-release-1.2/examples/guestbook/all-in-one/guestbook-all-in-one.yaml